src/v_video.c

```diff
@@ -215,28 +215,6 @@ void V_InitColorTranslation(void)
     }
 }
 
-void WriteGeneratedLumpWad(const char *filename)
```

Comment:
This was quite useful to extract color translation tables *after* they have been extended and adapted to the current palette in `V_InitColorTranslation()`, but I 'm not sure if people will miss it.